Documentation/mobile-manual-v3.txt

```diff
@@ -491,7 +492,226 @@ The summary page lists the date of your first dive in the dive list as well as y
 It also provides two columns of data for selectable time ranges (by default all dives in the dive list
 and the dives during the last six months.
 
-=== Export
+
+[[S_Stats]]
+== Dive Statistics
+
+[width="100%", frame="None"]
+|===
+a|image::mobile-images/StatsPanel.jpg["Image: Statistics panel",float="left"] |
+
+Creating meaningful statistics that convey the information you are looking for is surprisingly hard.
+Different users have very different needs and very different expectations. Subsurface-Mobile provides
+a rather detailed set of statistics features to summarise information from the dive log in a graphical
+way. To access the dive statistics tool, select _Main Menu_ -> _Statistics_. The statistics tool is
+so flexible that it is impossible to cover every possible use case. Rather, we provide a conceptual
+introduction, followed by specific examples. The tool can be closed using the _Back_ button (bottom
+right in Android).
+
+The Statistics tool has two panels (image on left):
+
+1. A setup panel containing comboboxes for requesting a graph (top).
+
+2. The graph that has been requested (bottom).
+
```

Comment:
whether it's top/bottom or left/right depends on the screen orientation. But I guess as you refer to the screenshot this still makes sense